application/models/Execution.php

```diff
@@ -23,6 +23,16 @@ function find($id)
         return $this->db->query("SELECT * FROM $this->table WHERE id = ?;", [$id])->row();
     }
 
+    public function insert($data)
```

Comment:
phpDoc 🎶

Reply:
Actually doing it on every method... :'(